2024.4

frenck opened the pull request:
# ⚠️ Do not merge. This PR is for reviewing purposes.

Augar committed:
Add Ecovacs goat switches (#31665)

c0ffeeca7 committed:
Views intro page: update (#31674)

- Update steps to refelct current software
- Add list introducing the different view types
- Addresses feedback from #31589

snoack committed:
Update Balboa Spa integration documentation for new fan and lights support (#31587)

c0ffeeca7 committed:
Dashboards: add page on sections view (#31691)

* Dashboards: add page on sections view

* Add note about grouping

* Rephrase

* Add yaml

* Update source/_includes/asides/dashboards_navigation.html

Co-authored-by: Klaas Schoute <hello@student-techlife.com>

* Update source/_dashboards/sections.markdown

---------

Co-authored-by: Klaas Schoute <hello@student-techlife.com>

catsmanac committed:
Document grid import / export and energy dashboard use for Enphase Envoy. (#31491)

* EnphaseNet

* Update text for disabled sensors.

* Tiny tweaks

* Spelling correction and 1 more abbreviation.

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Assist: add examples of new builtin sentences (#31703)

matrixd2 committed:
Update yolink supported device list (#31657)

jpbede committed:
Move Ping binary sensor attributes to sensor entities (#31705)

* Move Ping binary sensor attributes to sensor entities

* Fix

* Add note about disabled entities

* Update source/_integrations/ping.markdown

---------

Co-authored-by: G Johansson <goran.johansson@shiftit.se>

starkillerOG committed:
Add Reolink package detection docs (#31719)

starkillerOG committed:
Add Reolink PTZ patrol docs (#31721)

* Add Reolink PTZ patrol docs

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

karwosts committed:
Document energy devices detail card (#31559)

* Update energy.markdown for new detail devices card

* Add files via upload

* Rename devices-detail-graph.png to source/images/dashboards/energy/devices-detail-graph.png

* Update energy.markdown

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

yozik04 committed:
Add date platform and improve other text of Vallox integration (#31609)

* Add button platform and improve other text.

* cahange button to date

* PR review

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

jbouwh committed:
Remove deprecated `color_mode` flag, adjust `brightness` description for MQTT json light (#31707)

* Remove deprecated `color_mode` flag, adjust `brightness` description

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

IsakNyberg committed:
add is charging sensor docs (#31482)

* add is charging sensor docs

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Views: add illustration of layout strategies (#31745)

- illustrating the 3 traditional view types
- rephrase section on adding view so that it reads procedural
- add related topics

marcolettieri committed:
Added button platform to microBees integration (#31569)

* added button platform to microbees integration

* Update microbees.markdown

* Update source/_integrations/microbees.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

emontnemery committed:
Document Lutron Homeworks config flow (#31735)

* Document Lutron Homeworks config flow

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

timmo001 committed:
Add (#31734)

Noltari committed:
Airzone Cloud is now Cloud Push (#31728)

karwosts committed:
Update energy.markdown for detail devices max_devices (#31731)

frenck committed:
Merge branch 'current' into next

Bre77 committed:
Add sensors to Teslemetry (#31125)

* Add sensors to Teslemetry

* Update teslemetry.markdown

* Add disabled by default notes

* Add sensor metadata

belangp committed:
Update Hyperion documentation to include new effect sensor (#29429)

* Update Hyperion documentation to include new effect sensor

* Add clarity to hyperion sensor description

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

starkillerOG committed:
Add Reolink play quick reply message docs (#31722)

chamberlain2007 committed:
Add Aprilaire integration (#27933)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

emontnemery committed:
Document changes in integration_entities (#31697)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

starkillerOG committed:
Add Krispol as supported brand to Reolink (#31595)

RoboMagus committed:
Issues jinja template (#31712)

fwestenberg committed:
Add Beaufort windspeed to sensor markdown (#30356)

Co-authored-by: Erik Montnemery <erik@montnemery.com>

frenck committed:
Merge branch 'current' into next

chemelli74 committed:
Add Comelit humidifier/dehumidifier info (#31581)

frenck committed:
Merge branch 'current' into next

emontnemery committed:
Revert "Remove JuiceNet integration" (#31785)

shaiu committed:
seventeentrack update docs with config flow (#31579)

* seventeentrack update docs with config flow

* seventeentrack update docs with config flow

removed too much

* Update source/_integrations/seventeentrack.markdown

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

---------

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

jbouwh committed:
Add comment to mqtt sensor last_reset_value_template that `state_class` option must be `total` (#31844)

definitio committed:
Add A6 family code for DS2438 1-Wire sensor (#31816)

* Add A6 family code for DS2438 1-Wire sensor

* Apply review suggestion

Co-authored-by: epenet <6771947+epenet@users.noreply.github.com>

---------

Co-authored-by: epenet <6771947+epenet@users.noreply.github.com>

emontnemery committed:
Document binary_sensor entities (#31764)

emontnemery committed:
Document button entities (#31750)

timmo001 committed:
Add processes services to System Bridge (#29739)

* Add processes services to System Bridge

* Update system_bridge.markdown

* Update system_bridge.markdown

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

tronikos committed:
Sacramento Municipal Utility District (SMUD) docs (#31874)

* Create smud.markdown

* Update opower.markdown

jesserockz committed:
Add date to esphome platform list (#31771)

Commit:
Add docs for UniFi Protect remove_privacy_zone (#31598)

MatthiasLohr committed:
updated tolo documentation (#31882)

* updated tolo documentation

* small improvements

MatthiasLohr committed:
added ability to control aroma therapy slot (#31885)

thecode committed:
Remove OpenCV integration (#31888)

cosimomeli committed:
Add documentation for Ring Intercom (#31274)

Bre77 committed:
Add sensors to Teslemetry (#31767)

janiversen committed:
Modbus, configuration without entities (#31893)

MarkGodwin committed:
Docs for WAN port connection control to TP-Link Omada gateways (#31619)

* Update docs for TP-Link Omada PR 111431

* Remove non-docs.

* Doc consistency

bdraco committed:
Document profiler.set_asyncio_debug service (#31889)

* Document profiler.set_asyncio_debug service

* Update source/_integrations/profiler.markdown

dontinelli committed:
Initial documentation for fyta integration (#31480)

* Initial documentation for fyta integration

* Update source/_integrations/fyta.markdown

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

* Update source/_integrations/fyta.markdown

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

* Update source/_integrations/fyta.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/fyta.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/fyta.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/fyta.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_integrations/fyta.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update fyta.markdown

* Update fyta.markdown

Adjustment of sensor list to final PR, adjustment of unit for light

* Update fyta.markdown

Add explanations to units

---------

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

crug80 committed:
Update documentation to reflect modBusTTRegisters changes  (#30616)

* Update documentation to reflect modBusTTRegisters changes (possibility to define different setpoint registers belonging to HVAC Mode

* Clarified misunderstanding beetwen values and registers as requested by the maintainer

* Updated doc to reflect code changes

* Simplified the description as suggested

* Update source/_integrations/modbus.markdown

Co-authored-by: jan iversen <jancasacondor@gmail.com>

---------

Co-authored-by: jan iversen <jancasacondor@gmail.com>

Xitee1 committed:
Update octoprint.markdown (#31489)

Add 3 new Octoprint buttons to docs

boswelja committed:
Add documentation for Starlink Sleep configuration (#31343)

erwindouna committed:
Adding configflow documentation (#31943)

* Adding configflow documentation

* tiny style tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Thomas55555 committed:
Add device_tracker platform for husqvarna_automower (#31615)

nalin29 committed:
Add reference to new Room Audio Control (#31910)

Noltari committed:
Document AEMET diagnostics (#31583)

ludeeus committed:
Add sensors to the Traccar Server documentation (#31624)

* Add sensors to the Traccar Server documentation

* Update YAML examples

* Tiny style tweaks

* Update source/_integrations/traccar_server.markdown

* Add name examples

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

ludeeus committed:
Revert "Add sensors to the Traccar Server documentation (#31624)" (#31958)

This reverts commit b74807344d57dde19906993be5a731b3f5e2f650.

mill1000 committed:
Add fan preset modes feature (#31191)

* Update preset mode feature

* Update features documentation to reflect that fan preset modes are distinct from climate preset modes

* Add screenshot of fan preset modes

Bre77 committed:
Add Presets to Advantage Air (#31209)

* Add presets documentation for Advantage Air

* Fix separately

Bre77 committed:
Update tessie.markdown (#31450)

Noltari committed:
airzone_cloud: document Air Quality sensors (#31535)

* airzone_cloud: document Air Quality sensors

Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>

* tiny tweak

---------

Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

elupus committed:
Add event platform to rfxtrx docs (#31636)

* Add event mention to rfxtrx

* Update source/_integrations/rfxtrx.markdown

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

* Revert suggestion

* Revert suggestion

---------

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

Noltari committed:
airzone_cloud: document Air Quality binary_sensor (#31531)

* airzone_cloud: document Air Quality binary_sensor

Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>

* Update source/_integrations/airzone_cloud.markdown

Co-authored-by: Dave T <17680170+davet2001@users.noreply.github.com>

* airzone_cloud: remove extra whitespaces

Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>

---------

Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
Co-authored-by: Dave T <17680170+davet2001@users.noreply.github.com>

brenank committed:
Add statistics keep_last_sample option (#26348)

* introduce preserve last value option

* rename preserve_last_val to keep_last_sample

Bre77 committed:
Adds sensor entity in Tessie (#30881)

* Update tessie.markdown

* Update tessie.markdown

jesserockz committed:
Add time to esphome platform list (#31953)

Thomas55555 committed:
Add binary sensor platform to Husqvarna Automower (#31944)

* Add binary sensor to Husqvarna Automower

* fix

* tiny tweaks

* Rephrase battery charging

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

emontnemery committed:
Document state.last_reported (#31983)

* Document state.last_reported

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

starkillerOG committed:
Add Reolink HDD/SD storage sensor (#31509)

zweckj committed:
Add lamarzocco bluetooth docs (#31987)

* add bt docs

* Update lamarzocco.markdown

* Tiny tweaks

* Update lamarzocco.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Thomas55555 committed:
Add select platform to Husqvarna Automower (#31975)

chemelli74 committed:
Shelly add custom TCP port support (#31902)

* Shelly custom TCP port

* Tiny tweak

---------

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

wujiaqi committed:
Add supported services for lutron caseta blinds (#31957)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

marcolettieri committed:
Added binary sensors to microBees Integration (#31577)

MatthiasLohr committed:
improve TOLO integration documentation (#32008)

* improve TOLO integration documentation

* use - as marker for unordered lists

* fixed typo

* tiny tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

marcolettieri committed:
Added cover platform to microBees Integration (#31568)

* added cover to microBees Integration

* Update microbees.markdown

* Update microbees.markdown

* cover status microbees.markdown

catsmanac committed:
Add home battery storage entities for enphase_envoy (#32010)

* Add home battery storage entities for enphase_envoy

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

angelnu committed:
[thethingsnetwork] Update the to v3 (#31930)

* update documentation to v3

* Use step numbers in procedure

* Apply suggestions from code review

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

edenhaus committed:
Add Ecovacs event entities (#31968)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

malkstar committed:
Remove proxy_params config option from telegram bot platforms (#31798)

* Update telegram_polling.markdown

Remove proxy_params option from telegram bot

* Update telegram_webhooks.markdown

Remove proxy_params option from telegram bot

* Update telegram_broadcast.markdown

Remove proxy_params option from telegram bot

timmo001 committed:
Add camera in use and pending reboot binary sensors to System Bridge (#29876)

* Add camera in use and pending reboot binary sensors to System Bridge

* Update system_bridge.markdown

matrixd2 committed:
Update list of supported device models (#32024)

DCSBL committed:
Remove note about unavailable features for HomeWizard Watermeter (#32040)

LennP committed:
Add Motionblinds BLE integration documentation (#31224)

* Add MotionBlinds BLE integration documentation

* Update CODEOWNERS

* Update release

* Update Motionblinds name

* Update release

* tiny tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

frenck committed:
Allow numeric state trigger/condition against zone entities (#32038)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

synesthesiam committed:
Add docs for ollama_conversation (#31994)

* Re-add

* Add related topics

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Revert "[thethingsnetwork] Update the to v3" (#32042)

frenck committed:
Merge branch 'current' into next

LennP committed:
Add select platform to Motionblinds BLE (#32047)

brg468 committed:
Add smart hose timer info to Rachio (#30811)

* add smart hose timer info

* Add missing word

* Add notes about polling

* Update start watering service

* Tiny tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

LennP committed:
Add button to documentation (#32045)

GidoHakvoort committed:
Update rova documentation with config flow (#31941)

* Update rova documentation with config flow

* Tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

Ernst79 committed:
Update xiaomi_ble documentation (#31571)

msavazzi committed:
Added JVC Projector Sensor (#31077)

* Added documentation for JVC Projector Binary Sensor

* Update jvc_projector.markdown

* Update jvc_projector.markdown

* Tiny tweak

* Added JVC Project Sensor

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
fix ha release versions (#32059)

* FYTA: fix ha release version

* Motionblinds BLE: Fix typo in version number

the-ress committed:
List Prusa MK2.5/MK3 as supported (#32031)

* List Prusa MK2.5/MK3 as supported

* Document obtaining credentials on Prusa MK2.5/MK3

* tiny style tweak

* Update source/_integrations/prusalink.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

jbouwh committed:
Update humidity option type to float (#32056)

* Update humidity option type to float for mqtt

* Update humidity option type to float for generic hygrostat

frenck committed:
2024.4: Beta release notes

Quentame committed:
`conditional` & `entity-filter`: add ability to filter through `entity_id` & add `entity-filter` `conditional`'s conditions (#30561)

* entity-filter: add ability to filter through entity_id value

* Update to latest frontend PR edits

* Update to new upstream

* Fix English spelling

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

shaiu committed:
Fix Rova double header (#32097)

LennP committed:
Add more info to Motionblinds BLE integration (#32058)

* Add extra information for Motionblinds Bluetooth motors

* Fix Matter link

* Fix commas and period

* Fix spelling error

* Remove seller info

frenck committed:
Add floor template functions (#32121)

* Add floor template functions

* Add missing full stop

frenck committed:
Add label template functions (#32122)

* Add label template functions

* tiny tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

frenck committed:
2024.4: Sync codebase with docs

frenck committed:
2024.4: Add full changelog

frenck committed:
Add floor selector (#32124)

* Add floor selector

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/_docs/blueprint/selectors.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Map dashboard update to reflect current software (#32126)

frenck committed:
Add label selector (#32129)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

frenck committed:
2024.4: Update dev blogs links

frenck committed:
2024.4: Finish backward-incompatible changes

c0ffeeca7 committed:
Glossary: add term label (#32130)

c0ffeeca7 committed:
Glossary: add term definition of floor (#32128)

c0ffeeca7 committed:
Areas: add floors to term definition (#32132)

frenck committed:
2024.4: Finish new integrations section

frenck committed:
2024.4: Finish other noteworthy changes

frenck committed:
2024.4: Commit current status

c0ffeeca7 committed:
Glossary: add category (#32137)

Co-authored-by: Franck Nijhof <git@frenck.dev>

LennP committed:
Rename Motionblinds BLE integration to Motionblinds Bluetooth (#32117)

frenck committed:
2024.4: Finish textual content of release notes

frenck committed:
Merge branch 'current' into rc

janiversen reviewed (dismissed):
LGTM, thanks.

frenck committed:
2024.4: Date & title adjustments

frenck committed:
2024.4: Add intro

frenck committed:
2024.4: Link up categories term

c0ffeeca7 committed:
Organizing add section in docs (#32141)

* Docs: add section on organizing

* Adding floors and areas

* Fix typo

* Apply suggestions from code review

Co-authored-by: Franck Nijhof <git@frenck.dev>

* Add link to floor procedure

* Add links to templates

---------

Co-authored-by: Franck Nijhof <git@frenck.dev>

edenhaus committed:
Fix floor template function docs (#32143)

frenck committed:
2024.4: Add screenrecording for unlatching a lock

frenck committed:
2024.4: Add screenshots adding a shared Matter device

frenck committed:
2024.4: Add screenshots limiting the number of columns sections view

frenck committed:
2024.4: Add screenshots limiting the number of columns sections view

frenck committed:
2024.4: Add screenshots embedding webpage

joostlek commented on source/images/blog/2024-04/embeded-home-assistant-website.png:
embedded*

joostlek commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,766 @@
+
+### Item grouping
+
+Browsing a list of hundreds of {% term automations %}  or thousands of 
```
```suggestion
Browsing a list of hundreds of {% term automations %} or thousands of 
```

frenck committed:
2024.4: Improve embed webpage screenshot

frenck committed:
2024.4: Add screenshot map dashboard

c0ffeeca7 committed:
Add and apply labels (#32147)

* Add and apply labels

- add procedures on adding and applying labels
- update glossary entry to include scripts as label target

* Add missing screenshot

* Typo

---------

Co-authored-by: Franck Nijhof <git@frenck.dev>

edenhaus committed:
Revert "Fix floor template function docs" (#32146)

frenck committed:
2024.4: Add screenshots for new tables

frenck committed:
2024.4: Add filter panel screenshot

frenck committed:
2024.4: Update group items screenshot by manufacturer

frenck committed:
2024.4: Add floors screenshots

c0ffeeca7 committed:
Add categories (#32148)

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+---
+layout: post
+title: "2024.4: Organize all the things!"
+description: "Time for spring cleaning 🧹 Introducing new ways to organize your automation (and more) with floors, labels, and categories. Easily add Matter devices from other controllers, and two brand new map and webpage dashboards."
```
```suggestion
description: "Time for spring cleaning 🧹 Introducing new ways to organize your automations (and more) with floors, labels, and categories. Easily add Matter devices from other controllers, and two brand new map and webpage dashboards."
```

frenck committed:
2024.4: Add labels screenshot

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+easier to browse. It gives you a new perspective on organizing your vast
+network of devices.
+
+<img class="no-shadow" src='/images/blog/2024-04/group-items-by-manufacturer.png' alt='Screenshots showing the devices pages, grouped by manufacturer.'>
```
```suggestion
<img class="no-shadow" src='/images/blog/2024-04/group-items-by-manufacturer.png' alt='Screenshots showing the devices page, grouped by manufacturer.'>
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+
+Upon reading through and ideating on the feature request thread on
+[“Automation Grouping”](https://community.home-assistant.io/t/grouping-automations-on-frontend-for-organization/102771),
+we realize that organization can be done in multiple ways for different purposes.
```
```suggestion
we realized that organization can be done in multiple ways for different purposes.
```

c0ffeeca7 replied:
```suggestion
we realized that organization can be done in multiple ways for different purposes.
```

frenck committed:
2024.4: Add categories screenshot

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+
+{% term Floors %} and {% term areas %} are great for representing your actual
+home, but as everyone has their own ways of organizing their home, everyone
+has their own ways of organizing the Home Assistant, too!
```
```suggestion
has their own ways of organizing their Home Assistant, too!
```

frenck committed:
2024.4: Add uv developer blog

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+It sounds like a lot, but they all serve a distinct purpose. We are extending
+and clarifying {% term areas %} with {% term floors %}, and we are introducing
+{% term labels %} and {% term categories %} to make your {% term automations %}
+and many other tables to look more organized and to be more manageable.
```
```suggestion
and many other tables look more organized and be more manageable.
```

frenck committed:
2024.4: Add uv developer blog

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+integration to a map dashboard.
+
+You don't like having the map in your sidebar? Well, that is no longer a
+problem. Since it is now a dashboard it means you can remove in the dashboard
```
```suggestion
problem. Since it is now a dashboard it means you can remove it in the dashboard
```

frenck committed:
2024.4: Process review comments

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+This could be a web page from the internet or a local web page from a local
+machine or device like your router or NAS.
+
+<img class="no-shadow" src='/images/blog/2024-04/dashboard-webpage.png' alt='Screenshots adding a new webpage dashboard to Home Assistant, embeding the Home Assistant website.'>
```
```suggestion
<img class="no-shadow" src='/images/blog/2024-04/dashboard-webpage.png' alt='Screenshots showing addition of a new webpage dashboard to Home Assistant, embedding the Home Assistant website.'>
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+existing panels configured in your YAML configuration, Home Assistant will
+automatically migrate them to the new webpage dashboard on upgrade.
+
+<img class="no-shadow" src='/images/blog/2024-04/embeded-home-assistant-website.png' alt='Screenshot showing the Home Assistant website embeded into the Home Assistant frontend using a webpage dashboard.'>
```
```suggestion
<img class="no-shadow" src='/images/blog/2024-04/embeded-home-assistant-website.png' alt='Screenshot showing the Home Assistant website embedded into the Home Assistant frontend using a webpage dashboard.'>
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+are working on making the section view even better in the upcoming releases.
+
+In this release, based on your feedback, we have added the ability for you to
+define the maximum number of columns in the section view uses. This allows you
```
```suggestion
define the maximum number of columns that the section view uses. This allows you
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+- The [HomeWizard Energy] integration now supports the newest generation of
+  HomeWizard Energy Sockets! Thanks, [@DCSBL]!
+- We can now ship new releases of Home Assistant in waaay less time.
+  Building and shipping Home Assistant for you, when down from 1.5-2 hours to
```
```suggestion
  Building and shipping Home Assistant for you, went down from 1.5-2 hours to
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+{% details "MQTT" %}
+
+Support for `brightness` is now assumed for `mqtt` lights with the `json` schema
+unless only coelo modes `rgb`, `rgbw`, or `rgbww` are supported.
```
```suggestion
unless only color modes `rgb`, `rgbw`, or `rgbww` are supported.
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+For light which only support  color modes `rgb`, `rgbw`, or `rgbww`, 
+the `brightness` option flag can still be set to `false` if brightness is not
+supported, in this case brightness support is emulated in Home Assistant by
+scaling the RGBx-values.
```
```suggestion
For lights which only support  color modes `rgb`, `rgbw`, or `rgbww`, 
the `brightness` option flag can still be set to `false` if brightness is not
supported. In this case brightness support is emulated in Home Assistant by
scaling the RGBx-values.
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+If you have a Nibe SMO 20 in your account, you will notice that some 550
+entities are not provided by the integration.
+
+You can safely remove these entities as the integration does not use them.
```
```suggestion
entities are no longer provided by the integration.

You can safely remove these entities as the integration does not use them.
```

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -0,0 +1,773 @@
+The following integrations are also no longer available as of this release:
+
+- **OpenCV** has been removed. The integration has been in a broken state
+  for over an year and currently has no path forward.
```
```suggestion
  for over a year and currently has no path forward.
```

balloob committed:
Ollama: mention accessiblen etwork

MartinHjelmare commented on source/_integrations/ollama.markdown:
```diff
@@ -16,7 +16,7 @@ The **Ollama** {% term integration %} adds a conversation agent in Home Assistan
 This conversation agent is unable to control your house. The Ollama conversation agent can be used in automations, but not as a [sentence trigger](/docs/automation/trigger/#sentence-trigger). It can only query information that has been provided by Home Assistant. To be able to answer questions about your house, Home Assistant will need to provide Ollama with the details of your house, which inclu ...
 
-This integration requires an external Ollama server, which is available for macOS, Linux, and Windows. Follow the [download instructions](https://ollama.com/) to install the server.
+This integration requires an external Ollama server, which is available for macOS, Linux, and Windows. Follow the [download instructions]([https://ollama.com/](https://ollama.com/download)) to install the server. Once installed, configure Ollama to be [accessible over the network](https://github.com/ollama/ollama/blob/main/docs/faq.md#how-can-i-expose-ollama-on-my-network).
```
The download instructions link syntax looks incorrect.

frenck committed:
2024.4: Process review comments

frenck committed:
Fix ollama download link

MartinHjelmare commented on source/_posts/2024-04-03-release-20244.markdown:
```diff
@@ -121,7 +121,7 @@ types on the left that stay put and view the filtered results on the right.
 {% term automations %} can now be filtered not only by {% term areas %},
 {% term entities %} , or {% term devices %}, but also by used
-{% term blueprints %}, labels, and categories (more on these later!).
+{% term blueprints "blueprints," %} labels, and categories (more on these later!).
```
Is the double blueprints correct?

frenck replied:
Yes, it denotes the term to look up vs what to print on the page.

In this case, the `,` gets lost with an additional new-line on a desktop. Not sure why, never seen it before. This works around it.

frenck replied:
But it doesn't work so got to revert it :)

c0ffeeca7 committed:
Update page on organizing (#32149)

frenck committed:
2024.4: Process review comments
